ui/app/src/i18n/zh.json

```diff
@@ -84,6 +84,9 @@
       "jvm": {
         "title": "JVM 故障"
       },
+      "block": {
+        "title": "块存储 故障"
```

Comment:
There should be no space between the two Chinese phrases.

Reply:
@YangKeao Do you think this Chinese translation is okay?

Reply:
@g1eny0ung  I've already fixed it, thx !